static/compiler-service.js

```diff
@@ -31,7 +31,7 @@ var Promise = require('es6-promise').Promise;
 
 function CompilerService() {
     this.base = window.httpRoot;
-    if (!this.base.endsWith('/')) {
```

Comment:
Probably better to use _.endsWith, right? Im not sure how this would behave for empty bases

Reply:
yes, but i don't think that exists, i saw it's in lodash, but its not in underscore, and i think we're using underscore?